### PR TITLE
Issue 2 user view entry

### PR DESCRIPTION
**Pull request by vkahlon**
no entries

https://user-images.githubusercontent.com/47346471/147174873-5e330513-a164-4319-81f1-dee8d7e52f32.mp4

https://user-images.githubusercontent.com/47346471/147174881-60ba6160-49e2-4d81-9590-3cc790c55e92.mp4

https://user-images.githubusercontent.com/47346471/147174890-28d969cc-54bb-4d20-9f01-fe0553ee1999.mp4

https://user-images.githubusercontent.com/47346471/147174897-3eb5bf95-200b-4a4c-b2f9-6e8438c9f218.mp4

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -17,12 +17,88 @@ function retrieveEntryInfo(event) {
...
+  createParagraphElement.setAttribute('class', 'view-note-entry');
+  createParagraphElement.textContent = entry.message;
+  createSecondLiElement.appendChild(createParagraphElement);
+  return createDivElement;
+}
+function createTempJournal(object) {
+  var $theGrandDivTemp = document.querySelector('.temp-entry');
+  var tempEntry = createEntryTree(object);
+  $theGrandDivTemp.appendChild(tempEntry);
+}
```
Comment (codingcodymiller) [inline js/main.js:69]: What is the purpose of this function?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: js/main.js (hunk 1 above)
Comment (vkahlon) [reply js/main.js]: Changed the name of the function. To better suit its purpose.

**Comment by vkahlon** [comment]
Updated the var and element names with the comments to match what's going in that function.

Essentially instead of doing everything over from scratch, I take the object that I created for the step 1 exercise, pass it as an argument within the submit button, and the displayInSessionEntry will take that object, and store it in the house. It works immediately, and loads on top of the previous node list. Once the page reloads, the object disappears (its memory goes away), however, in the previous function (one that loads when you hit submit), it was stored in local storage. So the function below takes that object from local storage and builds it using the same frame work.
0d67bdca96a4e73560e3b8762ae4775c
It gets placed on the session-prepend-only class

**Comment by vkahlon** [comment]
Patched, two things. On HTML changed the div into a UL, and removed the div element with the class attribute:"session-prepend-only." In JS utilized the prepend method to that ul element.

File: js/main.js [hunk 2]
```diff
@@ -17,12 +17,88 @@ function retrieveEntryInfo(event) {
...
 var $getInfoFromSubmission = document.querySelector('#get-entry-form');
 $getInfoFromSubmission.addEventListener('submit', retrieveEntryInfo);
+
+function createEntryTree(entry) {
+  var createDivElement = document.createElement('div');
+  createDivElement.setAttribute('data-view', 'entries');
+
+  var createUlElement = document.createElement('ul');
+  createUlElement.setAttribute('class', 'row');
+  createDivElement.appendChild(createUlElement);
```
Comment (codingcodymiller) [inline js/main.js:40]: Now that we've addressed the other issues with your DOM tree, this needs to be addressed as well. You should only be generating an `li` for each entry, not a full `ul` with an `li` in it.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**